### PR TITLE
add function-square icon

### PR DESCRIPTION
**Pull request by mittalyashu**
| Preview |
| --- |
| CleanShot 2021-05-01 at 15 16 09@2x |

### CODE REVIEW

**Comment by locness3** [comment]
Should name it `function-square` and add a variant without a square.

File: icons/square-function.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M22 4a2 2 0 00-2-2H4a2 2 0 00-2 2v16a2 2 0 002 2h16a2 2 0 002-2V4z" />
+  <path d="M9 11.237h2.848M9 18c1.934 0 2.848-.996 2.848-2.793v-3.97m0 0h2.813m-2.813 0V8.984c0-1.937 1.034-3.265 3.152-2.933" />
```
Comment (ericfennis) [inline icons/square-function.svg:13]:
```suggestion
  <rect x="3" y="3" width="18" height="18" rx="2" ry="2" />
```

**Review by ericfennis — CHANGES_REQUESTED** [review]
Rename it to `function-square`.

File: icons/square-function.svg (hunk 1 above)
Comment (mittalyashu) [reply icons/square-function.svg:13]: @ericfennis Why `width` and `height` in `rect` tag?

Comment (ericfennis) [reply icons/square-function.svg:13]: `width` & `height` are used by the rect svg element to define the width and height 😛 .  See [docs](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/rect)

**Comment by mittalyashu** [comment]
@ericfennis This change b7e0ee8 has broken the icon see the preview in file change section.

File: icons/square-function.svg [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M22 4a2 2 0 00-2-2H4a2 2 0 00-2 2v16a2 2 0 002 2h16a2 2 0 002-2V4z" />
+  <rect x="3" y="3" width="18" height="18" rx="2" ry="2" />
+</svg>
```
Comment (ericfennis) [inline icons/square-function.svg:14]: I don't know what happend, maybe this help?
```suggestion
<svg
  xmlns="http://www.w3.org/2000/svg"
  width="24"
  height="24"
  viewBox="0 0 24 24"
  fill="none"
  stroke="currentColor"
  stroke-width="2"
  stroke-linecap="round"
  stroke-linejoin="round"
>
  <rect x="3" y="3" width="18" height="18" rx="2" ry="2" />
  <path d="M9 17c1.93 0 2.85-1 2.85-2.8V9.99c0-1.93 1.03-3.26 3.15-2.93M9 11.24h5.66" />
</svg>
```

**Comment by mittalyashu** [comment]
LGTM

**Comment by locness3** [comment]
Please rename as asked.

**Review by ericfennis — APPROVED** [review]

**Pull request merged**